content/en/infrastructure/process/_index.md

```diff
@@ -118,7 +118,26 @@ datadog:
 
 [1]: https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml
 {{% /tab %}}
+{{% tab "ECS Fargate" %}}
 
+In order to collect processes, the Datadog Agent must be running as a container within the Task.
+
+To collect process information in ECS Fargate, add the `PidMode` term to the Task Definition and set it as follows:
```

Comment:
Looking at other tabs here (Linux, Docker, helm) they list configuration for the `process_config.process_collection enabled`, directly or via env vars, we should include this here since these are expected to be complete instructions to enable the Agent.

Reply:
Do you know what the config would look like in ECS? [This link](https://docs.datadoghq.com/integrations/ecs_fargate/?tab=awscli) shows how to configure the Datadog agent on ECS Fargate, but I don't see anything about enabling process collection specifically.

Reply:
Should we have a link to the AWS docs regarding to what PidMode is?

I just looked at the docs at: https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#other_task_definition_params

Also, AWS calls this a task definition parameter, maybe something like this?
```suggestion
To collect process information in ECS Fargate, add the `PidMode` parameter to the Task Definition and set it to `task` as follows:
```